lib/Mojo/Util.pm

```diff
@@ -75,6 +75,13 @@ our @EXPORT_OK = (
   qw(unquote url_escape url_unescape xml_escape xor_encode)
 );
 
+
+# redirecting to MonkeyPatch::monkey_patch for backward compatibility
+    no warnings 'redefine';
+sub monkey_patch {
```

Comment:
You already import it, why re-define it here?

Reply:
If it is imported via `use` then it's also defined in the Mojo::Util namespace, so you can export it.

Reply:
you should be able to re-export symbols, I don't think that should be a problem

Reply:
will do that then :)